src/umpire/ResourceManager.cpp

```diff
@@ -163,6 +177,18 @@ ResourceManager::initialize()
     m_allocators.emplace_front(std::move(host_allocator));
   }
 
+  {
+    std::unique_ptr<strategy::AllocationStrategy>
+      allocator{
+        //util::wrap_allocator<strategy::AllocationTracker>(
+          registry.makeMemoryResource("__umpire_internal_null", getNextId())};
```

Comment:
Just a suggestion, so take it or leave it, but this and the zero-byte string could be constexpr static char defined above to reduce the change of typing errors.